ts/store/reducers/backendStatus.ts

```diff
@@ -353,35 +351,66 @@ export const barcodesScannerConfigSelector = createSelector(
 );
 
 /**
- * Return the remote config for the PN feature.
+ * Return the remote config about PN enabled/disabled
+ * if there is no data, false is the default value -> (PN disabled)
  */
-export const PnConfigSelector = createSelector(
+export const isPnEnabledSelector = createSelector(
```

Comment:
Since this is a direct read with minor data creation, should we remove the `createSelector`?